spring-boot-project/spring-boot-docs/src/main/asciidoc/deployment.adoc

```diff
@@ -324,15 +330,16 @@ covering these topics more in detail.
 === Boxfuse and Amazon Web Services
 https://boxfuse.com/[Boxfuse] works by turning your Spring Boot executable jar or war
 into a minimal VM image that can be deployed unchanged either on VirtualBox or on AWS.
-Boxfuse comes with deep integration for Spring Boot and will use the information from your
+Boxfuse comes with deep integration for Spring Boot and uses the information from your
 Spring Boot configuration file to automatically configure ports and health check URLs.
 Boxfuse leverages this information both for the images it produces as well as for all the
-resources it provisions (instances, security groups, elastic load balancers, etc).
+resources it provisions (instances, security groups, elastic load balancers, and so on).
 
-Once you have created a https://console.boxfuse.com[Boxfuse account], connected it to your
-AWS account, and installed the latest version of the Boxfuse Client, you can deploy your
-Spring Boot application to AWS as follows (ensure the application has been built by
-Maven or Gradle first using, for example, `mvn clean package`):
+Once you have created a https://console.boxfuse.com[Boxfuse account], connected it to
+your AWS account, installed the latest version of the Boxfuse Client, and ensured that
+the application has been built by Maven or Gradle (by using, for example, `mvn clean
+package`), you can deploy your Spring Boot application to AWS with a command similar to
+the following command:
```

Comment:
Drop "command" here to avoid the repetition?

Reply:
I dislike leaving "following" without an object, but it is technically correct, so OK. Thanks.

Reply:
![tukagym](https://user-images.githubusercontent.com/519772/32238958-72730038-be26-11e7-9490-e56ac11415dc.jpg)

Reply:
😂